src/Sulu/Bundle/AdminBundle/Resources/js/containers/FormOverlay/FormOverlay.js

```diff
@@ -0,0 +1,155 @@
+// @flow
+import React from 'react';
+import {action, computed, observable} from 'mobx';
+import type {ElementRef} from 'react';
+import {observer} from 'mobx-react';
+import Overlay from '../../components/Overlay';
+import {translate} from '../../utils';
+import Snackbar from '../../components/Snackbar';
+import Loader from '../../components/Loader';
+import Form, {ResourceFormStore} from '../Form';
+import type {FormStoreInterface} from '../Form/types';
+import type {Size} from '../../components/Overlay/types';
+import formOverlayStyles from './formOverlay.scss';
+
+type Props = {|
+    confirmDisabled: boolean,
+    confirmLoading: boolean,
+    confirmText: string,
+    formStore?: ?FormStoreInterface,
+    loading: boolean,
```

Comment:
This seems not to be used. And should the `loading` state not be read from the [`formStore::loading`](https://github.com/sulu/sulu/blob/3dd8b83bccf488196dcd526ede19dda0e4b52932/src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/stores/AbstractFormStore.js#L105) property?

Reply:
it is used here: https://github.com/sulu/sulu/pull/5905/files#diff-916771f5f650152bc3fb03ed379df868f07312e27fabca909b31946d8aa8518bR138-R148

the `<Form>` container will handle the `formStore::loading` property for us. but in the case of the `ProfileFormOverlay`, we load the data by ourselves (because the profile does not have an id and we cannot use the `ResourceFormStore`). it would be nice to display a loader in the overlay in this case

Reply:
but i am okay with removing the prop for now and thank about this when fixing #5759 🙂

Reply:
Okay I understand, the MemoryFormStore seems to have no loading state and get the data from outside. Could think that given the MemoryFormStore a promise instead of data and that it has so a `.loading` state would be nice. Also that the `profile` can not be a resource is something which isn't good but we would there need to implement something like `resource_type`. Overall for me its currently okay to let the `loading` prop there.